docs/source/mayavi/tips.rst

```diff
@@ -512,13 +543,35 @@ alongside with the `visual` handy for object-movement animation, see
     :ref:`mlab-animating-data`
 
 
+Creating a movie from an animation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is easy to create a movie from either an animation or by playing a
+timeseries.  To do this on the UI, select the Mayavi Scene and
+navigate to the Movie tab and select the record checkbox.  After this,
+if one animates a timestep via the "play" checkbox referred to in
+:ref:`animating_a_timeseries`, then a stack of images will be created
+in the directory specified in the movie UI.  This will also happen if
+one runs an animation as discussed in :ref:`animating_a_visualization`.
+
+This can also be scripted with mlab for example as follows::
+
+   from mayavi import mlab
+   f = mlab.figure()
+   f.scene.movie_maker.record = True
+   mlab.test_mesh_sphere_anim()
+
+This will create a set of images, one for each step of the animation.
+The ``movie_maker`` instance is available on each created scene.
+
+
 Animating a series of images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Let's say you have a stack of PNG or JPEG files that are numbered
 serially that you want to animate on a Mayavi scene.  Here is a simple
 script (called ``img_movie.py``)::
```

Comment:
I am trying to test this branch locally.  Is this img_movie.py in the repository?